tools/fregrid_acc/fregrid_acc.c

```diff
@@ -758,19 +731,18 @@ int main(int argc, char* argv[])
   if(debug) print_mem_usage("After get_input_output_cell_area");
   /* currently extrapolate are limited to ntiles = 1. extrapolate are limited to lat-lon input grid */
   if( extrapolate ) {
-    int i, j, ind0, ind1, ind2;
-    if(test_case ) mpp_error("fregrid: extrapolate is limited to test_case is false");
     if(ntiles_in != 1) mpp_error("fregrid: extrapolate is limited to ntile_in = 1");
     /* check if the grid is lat-lon grid */
-    for(j=1; j<=grid_in[0].ny; j++) for(i=1; i<=grid_in[0].nx; i++) {
-        ind0 = j*(grid_in[0].nx+2)+i;
-        ind1 = j*(grid_in[0].nx+2)+1;
-        ind2 = 1*(grid_in[0].nx+2)+i;
+    for(int j=1; j<=grid_in[0].ny; j++) {
+      for(int i=1; i<=grid_in[0].nx; i++) {
+        int ind0 = j*(grid_in[0].nx+2)+i;
```

Comment:
same comment as above I think its better to declare these first outside the loop.